src/__tests__/arrayUtils.test.ts

```diff
@@ -0,0 +1,22 @@
+import { arraysIsEqual } from '../utils/arrayUtils';
+
+describe('objectIsEmpty', () => {
+    it('should return true if all values are equal in the arrays', () => {
```

Comment:
Takler den childs, eller kun primitives?
[1, 'val', 3, 4, { prop1: 1 prop2: { anotherChildProp: 3}}];

Reply:
Kan være lurt å navngi shallow eller deep alt ettersom ja

Reply:
Legger inn endringer for å takle childs :)

Reply:
Nå skal metoden takle alt av objekt og array verdier.